indico/modules/events/registration/templates/management/email.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'layout/base.html' %}
+{% from 'forms/_form.html' import form_header, form_rows, form_footer  %}
+
+{% block title %}
+    {% trans %}Send email{% endtrans %}
+{% endblock %}
+
+{% block subtitle %}
+    {% trans %}to selected people{% endtrans %}
```

Comment:
I wouldn't inherit this template from base and just put the form in there. Combined with a title on the ajaxDialog it probably looks better that way